### PR TITLE
Update STaaS.md for cost estimation

### PR DESCRIPTION
**Pull request by julienVerm**

### CODE REVIEW

**Review by GuillaumeVadeBe — APPROVED** [review]

File: user-guide/Advanced_Functionality/Databases/STaaS/STaaS.md [hunk 1]
```diff
@@ -146,15 +146,38 @@ To request a cost estimation, follow the procedure below:
...
+      This way you will have the most accurate estimation of your system when running the script.
+
+   1. Click *Start migration*.
 
 1. Let the script run for 24 hours without restarting the DataMiner System (DMS).
 
 1. After the 24-hour period, restart the DMS to stop the estimation process.
 
-1. Notify your Skyline representative or email <staas@dataminer.services> to complete the process.
+1. The next morning you will be able to view your cost estimation in the [Admin](https://admin.dataminer.services) *Usage* module under the organization which the DMS is a part of.
```
Comment (MariekeGO) [inline user-guide/Advanced_Functionality/Databases/STaaS/STaaS.md:175]: This is a bit vague. Is this immediately after the 24-hour period? Is it some time during the morning after this period, using CE(S)T time? As our users are located all over the world, we may need to be more precise.

**Review by MariekeGO — COMMENTED** [review]
I've made some changes to the procedure based on the readme for the script, but I also noticed some possible issues (see other comments). Note also that it might be better to make sure that there is only a single source of truth for this procedure, instead of having duplicate info on the readme page and here. Ideally, the overlapping info on the readme page should be replaced with a link to this page.

File: user-guide/Advanced_Functionality/Databases/STaaS/STaaS.md [hunk 2]
```diff
@@ -146,15 +146,38 @@ To request a cost estimation, follow the procedure below:
...
-   At this point, an Automation script will be provided to you along with instructions on how to run it.
+1. Deploy the [STaaS Migration Script package](https://catalog.dataminer.services/details/46046c45-e44c-4bff-ba6e-3d0441a96f02) from the DataMiner Catalog.
+
+   On the Catalog page for this package, you can also find more information on how to run the script.
+
+1. In the Automation app in DataMiner Cube, locate the *CloudStorageMigration* script and [execute the script](xref:Manually_executing_a_script).
+
+1. Initialize the migration:
+
+   1. Optionally, configure a proxy. This is supported from DataMiner 10.4.6 onwards.
```
[the remark this answers is not in the document]
Comment (julienVerm) [reply user-guide/Advanced_Functionality/Databases/STaaS/STaaS.md:157]: using a proxy is supported from 10.4.5 but migrating data with proxy setup to staas is supported from 10.4.6

[the remark this answers is not in the document]
Comment (julienVerm) [reply user-guide/Advanced_Functionality/Databases/STaaS/STaaS.md:157]: As far as I can tell 10.4.7 should be 10.4.6. I will double check

File: user-guide/Advanced_Functionality/Databases/STaaS/STaaS.md [hunk 3]
```diff
@@ -146,15 +146,38 @@ To request a cost estimation, follow the procedure below:
...
+
+   1. Click *Init migration*.
+
+1. Start the migration:
+
+   1. Make sure *Replication only* is selected.
+
+   1. Select all data types.
```
[the remark this answers is not in the document]
Comment (MariekeGO) [reply user-guide/Advanced_Functionality/Databases/STaaS/STaaS.md:165]: Do we intend for users to do the migration themselves with this script? Right now, the documentation says to always contact Skyline for this.

File: user-guide/Advanced_Functionality/Databases/STaaS/STaaS.md (hunk 1 above)
Comment (MariekeGO) [reply user-guide/Advanced_Functionality/Databases/STaaS/STaaS.md:175]: Yes, we should definitely make sure it is clear when they can expect this; that way we prevent unnecessary questions.

File: user-guide/Advanced_Functionality/Databases/STaaS/STaaS.md (hunk 3 above)
[the remark this answers is not in the document]
Comment (julienVerm) [reply user-guide/Advanced_Functionality/Databases/STaaS/STaaS.md:165]: I asked the team and for now we do not want users to do migration themselves but they should keep contacting us first. I can update the readme so the note mentions this consideration is not needed for performing estimation

**Review by MariekeGO — APPROVED** [review]

**Pull request merged**